homeassistant/components/n26/sensor.py

```diff
@@ -204,7 +209,8 @@ def update(self) -> None:
     @property
     def unique_id(self):
         """Return the unique ID of the entity."""
-        return "space_{}".format(self._space["name"].lower())
+        return "space_{}_{}".format(self._data.balance["iban"][-4:],
+                                    self._space["name"].lower())
```

Comment:
This is a breaking change, please add that to the PR description (as per template).